README.md

```diff
@@ -26,7 +26,10 @@ brew install coreutils
 
 #### Optional
 * [fd](https://github.com/sharkdp/fd) a modern `find` replacement, it will use this preferentially if it's installed otherwise fallback to `find`
+* [pv](https://www.ivarch.com/programs/pv.shtml) a pipe viewer with which we can
+  limit the output speed to emulate the speed back in the days. :)
 
+  Set `ANSI_ART_RATE_LIMIT` to eg `8k` to limit data rate to 8192 bps.
```

Comment:
Nice, though can we set this to use the `ANSI_MOTD` ENV namespace to keep it consistent with the others eg. `ANSI_MOTD_RATE_LIMIT`